types/zengin-code/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [ "../"],
+        "types": [],
+        "noEmit": true,
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
It's for consuming a package, not used when definition on DT (unless there is a specific use case)